msdk-rawdata/msdk-rawdata-filters/src/test/java/io/github/msdk/rawdata/filters/MeanFilterMethodTest.java

```diff
@@ -119,7 +119,7 @@ public void testMeanFilter() throws Exception {
       float intensityAverage = inputScan.getTIC() / inputScan.getNumberOfDataPoints();
 
       for (float intValue : intensityValues) {
-        Assert.assertEquals(intensityAverage, intValue, 0.001);
+        Assert.assertEquals(intensityAverage, intValue, 0.4);
```

Comment:
So if you read with the old parser and with the new parser, and compare the arrays returned by getIntensityValues[], the values in the arrays are different?
I am sure you can find the cause if you dig a bit deeper into the issue, add some more debugging output, etc.